network/netcore/src/transport/tcp.rs

```diff
@@ -105,8 +109,29 @@ impl Transport for TcpTransport {
             .or_else(|| parse_dns_tcp(protos).map(|_| ()))
             .ok_or_else(|| invalid_addr_error(&addr))?;
 
+        let proxy_addr = match std::env::var("https_proxy") {
```

Comment:
Do we expect to be changing this on the fly?  Or can we load this on startup via the NetworkConfig?  Or at least just load the environment variable once on startup instead of every time.

Reply:
Now, I see this is loading an environment variable and always parsing it the same.

Reply:
No, it's not expected to change, but I figured reading environment variables should be pretty cheap and outgoing Libranet connections are not made that frequently.

Reply:
@gregnazario Are you okay with this? I actually can't see a good place to do this at startup since the `TcpTransport` is created in `const LIBRA_TCP_TRANSPORT`.

Reply:
Yeah, let's just do this for now.  Looks like the best way would probably to make a constructor, and load it where the `const` is originally made.